RadixWallet/Features/HomeFeature/Coordinator/Home.swift

```diff
@@ -372,6 +382,15 @@ public struct Home: Sendable, FeatureReducer {
 			await npsSurveyClient.uploadUserFeedback(feedback)
 		}
 	}
+
+	private func loadAccountResources() -> Effect<Action> {
```

Comment:
Accounts initial resources are loaded in one go, prefer to have a single point of observation for it, and then updated the rows, instead of having it per row.